operator/duties/sync_committee.go

```diff
@@ -199,7 +192,7 @@ func (h *SyncCommitteeHandler) fetchAndProcessDuties(ctx context.Context, period
 	}
 	lastEpoch := h.network.Beacon.FirstEpochOfSyncPeriod(period+1) - 1
 
-	allActiveIndices := indicesFromShares(h.validatorProvider.ParticipatingValidators(firstEpoch))
+	allActiveIndices := h.validatorController.AllActiveIndices(firstEpoch, waitForInitial)
```

Comment:
@olegshmuelov @moshe-blox is `ParticipatingValidators` not enough here?

Reply:
or maybe we still want to use `AllActiveIndices` because the `waitForInitial` logic.

Reply:
if you use it only for the `waitForInitial` logic maybe its best to separate it.